include/eosio/vm/execution_context.hpp

```diff
@@ -312,11 +312,11 @@ namespace eosio { namespace vm {
 
                   vm::invoke_with_signal_handler([&]() {
                      result = execute<sizeof...(Args)>(args_raw, fn, this, base_type::linear_memory(), stack);
-                  }, handle_signal);
+                  }, &handle_signal);
```

Comment:
I don't understand this either.  AFAIK, a function should be an lvalue and be deduced as an lvalue reference.  Does gcc behave differently depending on the standard version?  Well, this change does no harm in any case.

Reply:
Well, c++20/23 additionally errors out elsewhere. And can't go less than 17, so not much else to really probe :shrug: